scripts/startTriggerService.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export _JAVA_OPTIONS="-Xms8m -Xmx128m"
+
+set -e
+
+TRIGGER_SERVICE_HOST="127.0.0.1"
+TRIGGER_SERVICE_PORT=8088
+
+cleanup() {
+    pids=$(jobs -p)
+    echo Killing $pids
+    [ -n "$pids" ] && kill $pids
+}
+
+waitForTriggerService() {
+    until curl ${TRIGGER_SERVICE_HOST}:${TRIGGER_SERVICE_PORT}/livez
+    do
+        sleep 0.1
+    done
+}
+
+startTrigger() {
+    curl -X POST -H "Content-Type: application/json" \
+        -d "{ \"triggerName\": \"${1}\", \"party\": \"${2}\", \"applicationId\": \"my-app-id\" }" \
```

Comment:
does this work? reads better:
```suggestion
        -d "{ 'triggerName': '${1}', 'party': '${2}', 'applicationId': 'my-app-id' }" \
```

Reply:
No, the two are _not_ the same. The original produces a valid JSON while the suggestion does not. The following would do, however I think I'd prefer the escapes: `'{ "triggerName": "'"${1}"'", "party": "'"${2}"'", "applicationId": "my-app-id" }'`

Reply:
As Tomi told (including the preference for escapes).

Reply:
ok, didn't know. great JSON feature, that both 'x' and "x" are Javascript strings, but one of them is a JSON string :)

Reply:
JSON is not JavaScript. There are plenty other things that don't work in JSON, e.g. comments.

Reply:
then how about this?
```
payload=$(printf '{ "triggerName": "%s", "party": "%s", "applicationId": "my-app-id" }' "$1" "$2")
curl ... "$payload"
```